src/main/java/vg/civcraft/mc/civmodcore/itemHandling/NiceNames.java

```diff
@@ -4,45 +4,20 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import org.bukkit.Bukkit;
-import org.bukkit.Material;
 import org.bukkit.enchantments.Enchantment;
-import org.bukkit.inventory.ItemStack;
-import org.bukkit.inventory.meta.ItemMeta;
 
 public class NiceNames {
```

Comment:
Should adjust the name of this class if its only holding aliases for Enchantments

Reply:
NiceNames will be removed entirely within the next day with the addition of EnchantingAPI

Reply:
Sounds good